app/assets/stylesheets/pages/events.scss

```diff
@@ -154,10 +154,12 @@
   overflow: auto;
   .event-last-push-text {
     @include str-truncated(100%);
+    padding: 5px 0;
```

Comment:
Properties should be ordered float, font-size, line-height, margin-right, padding, padding-right